### PR TITLE
Milestone 2 - Action, Reducers, Store, and Components

### PR DESCRIPTION
**Pull request by JuanPabloGil**
On this Milestone was created the basic structure of the next features: Action, Reducers, Store, and Components.

[Live Preview](https://limitless-refuge-25854.herokuapp.com/)

### CODE REVIEW

File: src/components/BooksList.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {useSelector} from 'react-redux'
+import Book from './Book'
+
```
Comment (shloch) [inline src/components/BooksList.jsx:4]: please relocate this file into `src/containers ` folder

File: src/components/BooksForm.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
```
Comment (shloch) [inline src/components/BooksForm.jsx:2]: please relocate this file into `src/containers`  folder

File: src/components/BooksList.jsx (hunk 1 above)
Comment (Salvador-ON) [reply src/components/BooksList.jsx:4]: we relocate the files

File: src/components/BooksForm.jsx (hunk 2 above)
Comment (Salvador-ON) [reply src/components/BooksForm.jsx:2]: we relocate the files

**Pull request closed without merging**